_docs/example-collections.md

```diff
@@ -150,7 +150,7 @@ SET LIT first_name "Jane"
 SET LIT last_name "Doe"
 
 SET LIT phone_country "GH"
```

Comment:
I guess we want to remove the `phone_country` requirement as well, once we fully test our system is working without it

Reply:
Sure. I will proceed removing `phone_country` from all examples and ensure to test with QA 👍

Reply:
@sztupy this has now been addressed: https://github.com/transferzero/api-documentation/pull/135/commits/050e2cd927968b9a9f609111aa6f87db0aadbdae

I will wait to release everything to `production` before merging this PR.